### ISSUE TITLE
i think tiktok changed a few things.

### ISSUE THREAD

**Issue by JeyKul** [vantezzen/wrapped#7]
I cant open any file in the website.
I might be a bit of an tiktok addict but my userdata is 84mb in size.
and the insides are a total mess with different sections of the data being split and continued elsewhere. either tiktok fucked up the file orrr smth is new or wrong
![grafik](https://github.com/user-attachments/assets/347b8ba3-9245-412b-8073-e14f0ad1b78f)
![grafik](https://github.com/user-attachments/assets/37978625-5557-4189-a325-69fed15fe32b)

**Comment by vantezzen** [comment]
Thanks for reporting, I just requested a new export from TikTok just now - let's see if that will work

**Comment by vantezzen** [comment]
Ok I think this is just an issue with the size of the export. My data is around 32MB in size and still parses fine

**Comment by JeyKul** [comment]
> Ok I think this is just an issue with the size of the export. My data is around 32MB in size and still parses fine


hi, thank you so much for checking. do you want me to send you my data so you can maybe work something out? i would be happy to do so. But i dont want to share it in this issue

**Comment by vantezzen** [comment]
Yes that would be interesting - I don't know if it's possible to fix this easily if it's a browser-level issue but I'll take a look.
You can send it to hello@vantezzen.io (or maybe a Google Drive link or something if email's too small).
